staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/helpers.go

```diff
@@ -116,3 +118,76 @@ func CRDRemoveFinalizer(crd *CustomResourceDefinition, needle string) {
 	}
 	crd.Finalizers = newFinalizers
 }
+
+// GetJSONPath returns the value in the data at JSONPath path.
+func GetJSONPath(data interface{}, path string) interface{} {
```

Comment:
we have public unstructured helpers now. Would they work for you? apimachinery/pkg/apis/meta/v1/unstructured/helpers.go

Reply:
yes, they probably should. I'll try with them.

Reply:
Done.